requirements.txt

```diff
@@ -9,7 +9,8 @@ cryptography==0.9.1
 debtcollector==0.5.0
 docker-py==1.3.1
 effect==0.1a13
-eliot==0.8.0
+eliot==0.9.0
+eliot-tree==15.3.0
```

Comment:
This branch was originally intended to have Flocker specific documentation on reading the Eliot logs. That's why `eliot-tree` got added to the `requirements.txt`. But we moved documentation to a separate issue: https://clusterhq.atlassian.net/browse/FLOC-3173
I'll leave `eliot-tree` in there for now.